lib/providers/services/firebase/push_notification_service.dart

```diff
@@ -32,10 +36,7 @@ class PushNotificationService {
         },
         onResume: (Map<String, dynamic> message) async {
           print("onResume: $message");
-
-          final dynamic data = message['data'] ?? message;
-          print("onResume: data");
-          print("onResume: $data");
+          backgroundMessageHandler(message, context);
```

Comment:
Great... we can do link handling the same way, maybe? As a service that's always on?